examples/gadgets/runner/README.md

```diff
@@ -0,0 +1,24 @@
+# Runner example
```

Comment:
I think we need to have a guide in the `docs/guides` directory so it becomes available on:
https://inspektor-gadget.io/docs/latest/guides/

Then, `examples/gadgets/runner/README.md` can reference the official documentation URL above.